frontend/src/components/Resources/Resources.css

```diff
@@ -0,0 +1,158 @@
+@import url('http://fonts.cdnfonts.com/css/abeezee');
+
+*{
+    font-family: 'ABeeZee', sans-serif;
+}
+
+.resources-container{
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-evenly;
+    margin: 60px auto;
+    min-height: 800px;
+}
+
+.resources-upper-sidescroll{
+    /* display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-evenly; */
```

Comment:
Do we need these commented lines on the code?

Reply:
No, will be deleting those.